ramls/invoice.raml

```diff
@@ -64,6 +67,33 @@ resourceTypes:
         is: [validate]
       delete:
         description: Delete an invoice with given {id}
+
+      /documents:
```

Comment:
Please also use `validate` trait wherever useful...

Reply:
trait added

Reply:
I think better to add `validate` trait only to `POST`